spec/schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "An API to support lodging objections to compulsory and voluntary strike offs",
+    "version": "1.0.0",
+    "title": "Companies House - Strike Off Objections API"
+  },
+  "host": "api.companieshouse.gov.uk",
+  "basePath": "/company/{companyNumber}/objections/strike-off/request",
+  "schemes": [
+    "https"
+  ],
+  "tags": [
+    {
+      "name": "Strike Off Objections"
+    }
+  ],
+  "paths": {
+    "/test": {
```

Comment:
How about instead of /test, we just post to the basePath - /company/{companyNumber}/objections/strike-off/request
This could possibly then be the real 'create' endpoint
POST -> /company/{companyNumber}/objections/strike-off/request
and would eventually return the id of the request created. But for now just return some dummy id.

(Just thinking ahead) - Then we could have a /process endpoint which would also allow support to re-process requests if they modify them.

Reply:
Yep. Will amend.